units/patch_intent.go

```diff
@@ -308,7 +308,7 @@ func (j *patchIntentProcessor) finishPatch(ctx context.Context, patchDoc *patch.
 		var patchConfig *model.PatchConfig
 		repeatPatchID, shouldRepeat := j.intent.RepeatPreviousPatchDefinition()
 		if shouldRepeat {
-			patchDoc.ReferencePatchID = repeatPatchID
+			patchDoc.ReferenceManifestID = repeatPatchID
```

Comment:
To double check, the manifest `_id` is the same as the patch `_id`? If so, it would be worth calling out in a comment that they're one and the same, and this isn't a bug.

Reply:
Yes, if a manifest exists- it will be the same as the _id.